ts/DataGrid/DataGrid.ts

```diff
@@ -646,7 +640,7 @@ class DataGrid {
     private renderNoData(): void {
         makeHTMLElement('div', {
             className: Globals.classNames.noData,
-            innerText: 'No data to display'
+            innerText: this.options?.lang?.noData || ''
```

Comment:
There should be the default text instead of an empty string.

Reply:
I know, but what is the use case for an empty string? If default is defined, this OR does not make sense.